docs/getting-started.md

```diff
@@ -157,6 +157,31 @@ You can check out [this example project](https://github.com/henrikra/nativeNavig
 
 Remember that you need to wrap each screen that you use in your app with `gestureHandlerRootHOC` as with native navigation libraries each screen maps to a separate root view. It will not be enough to wrap the main screen only.
 
+### Usage with modals on Android
+On Android RNGH does not work by default because modals are not located under React Native Root view in native hierarchy.
+In order to make it workable, components need to be wrapped with `gestureHandlerRootHOC` (it's no-op on iOS and web).
+
+E.g. 
+```js
+const ExampleWithHoc = gestureHandlerRootHOC(
+  function GestureExample() {
+    return (
+      <View>
+        <DraggableBox />
+      </View>
+    );
+  }
+);
+
+export default function Example() {
+  return (
+    <Modal animationType="slide" transparent={false}>
+      <ExampleWithHoc />
+    </Modal>
+  );
+}
+``` 
+
```

Comment:
Just a question, would it be possible to export a modal from RNGH that already handles gestures as it should on Android?

Reply:
Good point!

Reply:
... but let's do it in follow-up